### PR TITLE
June2020 news and members

### PR DESCRIPTION
**Pull request by KorayKirli**

### CODE REVIEW

**Review by vinjana — COMMENTED** [review]
Only small things.

File: _includes/team_members.html [hunk 1]
```diff
@@ -16,6 +16,15 @@ <h2 data-i18n="team_members.title"></h2>
                 <p data-i18n="[html]team_members.member11"></p>
                 <p data-i18n="[html]team_members.member12"></p>
                 <p data-i18n="[html]team_members.member13"></p>
+                <p data-i18n="[html]team_members.member14"></p>
```
[the remark this answers is not in the document]
Comment (KorayKirli) [reply _includes/team_members.html:19]: We will switch to a new website soon, so, for the time being, I will leave them like this.

File: assets/locales/de/translation.json [hunk 2]
```diff
@@ -27,14 +27,34 @@
   "news": {
     "navTitle": "Neues",
     "title": "Neuigkeiten",
-    "item7": {
+    "item12": {
```
[the remark this answers is not in the document]
Comment (KorayKirli) [reply assets/locales/de/translation.json:30]: We will switch to a new website soon, so, for the time being, I will leave them like this.

File: assets/locales/en/translation.json [hunk 3]
```diff
@@ -30,29 +30,57 @@
...
+    "member21": "<b>Lisa Kaldowski</b>, Research Fellow - Universitätsklinikum Heidelberg",
+    "member22": "<b>Amelie Schenk</b>, Research Assistant -H eidelberger Akademie der Wissenschaften"
   },
   "news": {
     "navTitle": "News",
     "title": "News",
-    "item7": {
+    "item12": {
+      "title": "May 2021 - NFDI: New activities kicking off within the newly established NFDI E.V.",
+      "content": "GHGA is funded by the DFG via the German national program for research data infrastructures, the NFDI. Recently, the NFDI office in Karlsruhe has been turned into a registered association - a “eingetragener Verein” or e.V. in German. As part of the activities of NFDI e.V., GHGA will be involved in several cross-cutting activities, which are currently being formed as sections within <a href=\"https://www.nfdi.de/\" target=\"_blank\">NFDI e.V.</a>."
```
[the remark this answers is not in the document]
Comment (KorayKirli) [reply assets/locales/en/translation.json:58]: currently not, the author asked me to leave it as it is.

**Review by KorayKirli — COMMENTED** [review]
Changes suggested by @vinjana

**Review by vinjana — APPROVED** [review]

**Pull request merged**